pkg/sdk/python/channels.py

```diff
@@ -20,13 +20,13 @@ def channels(self, query_params, token):
         return resp.json()
 
     def channel_by_thing(self, thing_id, query_params, token):
```

Comment:
This returns a list of channels so better call it `channels_by_thing` or `list_chanels_by_thing`